spec/machine/ARM/Arch_Kernel_Config_Lemmas.thy

```diff
@@ -0,0 +1,19 @@
+(*
+ * Copyright 2023, Proofcraft Pty Ltd
+ *
+ * SPDX-License-Identifier: GPL-2.0-only
+ *)
+
+(* Architecture-specific lemmas constraining Kernel_Config definitions *)
+
+theory Arch_Kernel_Config_Lemmas
+imports
+  Kernel_Config_Lemmas
+  Setup_Locale
+  Platform
```

Comment:
@lsf37 Platform imports Setup_Locale, so we can drop the Setup_Locale (and squash that change too)

Reply:
I'll do that and pile it on top.